app/views/declaration/dispatch_location.scala.html

```diff
@@ -14,72 +14,84 @@
  * limitations under the License.
  *@
 
+@import controllers.navigation.Navigator
 @import forms.declaration.DispatchLocation
 @import forms.declaration.DispatchLocation.AllowedDispatchLocations._
-@import services.Countries
-@import uk.gov.hmrc.play.views.html._
-@import views.components.inputs.RadioOption
-@import views.Title
-@import controllers.navigation.Navigator
 @import models.requests.JourneyRequest
+@import services.Countries
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.components.gds.Styles._
+@import views.html.components.gds._
+@import views.{BackButton, Title}
 
-@this(main_template: views.html.main_template)
+@this(
+        govukLayout: gdsMainTemplate,
+        govukButton: GovukButton,
+        govukRadios: GovukRadios,
+        govukDetails : GovukDetails,
+        errorSummary: errorSummary,
+        sectionHeader: sectionHeader,
+        saveButtons: saveButtons,
+        formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+)
 
 @(mode: Mode, form: Form[DispatchLocation])(implicit request: JourneyRequest[_], messages: Messages)
 
-@tariffBodyDesc = { @Html(messages("declaration.type.dispatchLocationSummaryText", components.details_content_link())) }
-
-@tariffBody = {
-<p>
-    <span class="bold">@messages("supplementary.dispatchLocation.summary.euCountries") </span>
+@countriesDetails = {
+    <b>@messages("supplementary.dispatchLocation.summary.euCountries") </b>
     @Countries.euCountries.mkString(", ")
-</p>
-<p>
-    <span class="bold">@messages("supplementary.dispatchLocation.summary.euSpecialFiscalTerritories") </span>
+    <br><br>
```

Comment:
AFAIR we were usually using some class for this purpose. They were ending with sth like "...margin-9"

Reply:
It's a bit of a hack either way.  The prototype uses two 'govuk-details__text' divs but the GDS 'Details' component we are using doesn't seem to support it.